package.json

```diff
@@ -109,9 +112,9 @@
   "scripts": {
     "start": "webpack --config ./client/webpack.config.js --mode development --progress --watch",
     "build": "webpack --config ./client/webpack.config.js --mode production",
-    "fix:js": "eslint --ext .js,.ts,.tsx --fix ./",
+    "fix:js": "eslint --ext .js,.ts,.tsx --fix .",
     "format": "prettier --write '**/?(.)*.{css,scss,js,ts,tsx,json,yaml,yml}'",
-    "lint:js": "eslint --ext .js,.ts,.tsx --report-unused-disable-directives --max-warnings 16 ./client",
+    "lint:js": "eslint --ext .js,.ts,.tsx --report-unused-disable-directives .",
```

Comment:
Thanks for fixing this, gonna assume that was me sorry :) 

Could you please split out these fixes to a separate commit, just will be easier to look at in the history